### DISCUSSION TITLE
scripts/postinstall.js hook Error on npm install

### DISCUSSION THREAD

**Discussion by imadtouil** [prisma/prisma#2953] [Q&A]
Hi,

while building my app & running npm install, i can see that the postinstall hook is failing to locate the schema.prisma file.

`> @prisma/client@2.2.0 postinstall /usr/app/node_modules/@prisma/client`
`> node scripts/postinstall.js`

`Error: Either provide --schema or make sure that you are in a folder with a schema.prisma file.`

Also the documentation doesn't cover how to setup schema for hooks.

**Comment by janpio** [comment]
Do you have your schema in a special, non standard location?

**Reply by imadtouil** [reply]
Thanks @Jan, I'm using it under prisma/ the default generated folder when running `prisma init`

**Reply by imadtouil** [reply]
@janpio please note that i'm facing the same issue while using your code source example provided here https://www.prisma.io/docs/guides/deployment/deploying-to-vercel#notes

**Reply by ryands17** [reply]
Hey @imadtouil :wave: 
Could you share your repo so I can check it from my end?
